reconcile/change_owners/changes.py

```diff
@@ -454,3 +457,35 @@ def parse_bundle_changes(
             )
 
     return change_list
+
+
+def aggregate_resource_changes(
+    bundle_changes: list[BundleFileChange],
+    content_store: list[dict[str, Any]],
+) -> list[BundleFileChange]:
+    new_bundle_changes: list[BundleFileChange] = []
+    resource_changes: list[BundleFileChange] = []
+    for change in bundle_changes:
+        new_bundle_changes.append(change)
```

Comment:
instead of `append` to `new_bundle_changes` on every `bundle_changes`, can directly return `bundle_changes + resource_changes` as the final result at the end.

Reply:
i was being consistent with above methods